event_processor/event_processor/base/spider_base.py

```diff
@@ -32,7 +33,7 @@ def empty_check_extract(self, base_selector, extractor, path, default_value=''):
         return list( \
                 map(lambda data: \
                         default_value if len(data) == 0 \
-                        else data[0], \
+                        else " ".join(map(lambda x: self.normalize_whitespace(x), data)), \
```

Comment:
If a lambda takes the same arguments as the helper function, you can apply the helper function directly
```suggestion
                        else " ".join(map(self.normalize_whitespace, data)), \
```